test/integration/document-file-dependencies/test/index.test.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-env jest */
+
+import { join } from 'path'
+import {
+  fetchViaHTTP,
+  findPort,
+  killApp,
+  nextBuild,
+  nextStart,
+} from 'next-test-utils'
+import cheerio from 'cheerio'
+
+jest.setTimeout(1000 * 60 * 2)
+const appDir = join(__dirname, '..')
+
+let appPort
+let app
+
+describe('File Dependencies', () => {
+  describe('production mode', () => {
+    beforeAll(async () => {
+      appPort = await findPort()
+      await nextBuild(appDir)
+      app = await nextStart(appDir, appPort)
+    })
+
+    afterAll(() => killApp(app))
+
+    it('should depend on global and module css files in standard page', async () => {
```

Comment:
These tests shouldn't check for stylesheets being there, instead you can check if the styles are loaded correctly by reading the applied styling of different elements